### DISCUSSION TITLE
JIT feature idea: Arbitrary responsive settings

### DISCUSSION THREAD

**Discussion by let-lc** [tailwindlabs/tailwindcss#4100] [Ideas]
I think Tailwind's built-in responsive screen sizes are good enough for handling most cases. But while I was working on a new site today, I needed to hide a div in screen when the screen width is above 860px. I already had JIT enabled, so I thought maybe I can do something like `[860px]:hidden` instead of adding it in the tailwind config file, and it didn't work. Did some search but I couldn't find anything, so maybe this feature is not in JIT?

So my idea is having a class like `[min-w-<size>]:<class>` which can convert to `@media (min-width: <size>){...}`, or even `[max-w-<size>]:<class>` so it can also handles max-width like `@media (max-width: <size>)`, more flexibility.

The `@media` rule is big category. Maybe in future version, not just screen width, but also something like orientation or other media rule values. I think it will be great if we can have something like this in future version of Tailwind CSS.

**Comment by quick007** [comment]
You can always do this with your tailwind config

**Reply by let-lc** [reply]
I understand that as I mentioned above. This is an "ideas" category discussion. So I am just suggesting an idea and that's all, not looking for a solution. But thank you! ❤️
